include/dbg_macro/dbg.h

```diff
@@ -29,23 +29,36 @@ License (MIT):
 #ifndef DBG_MACRO_DBG_H
 #define DBG_MACRO_DBG_H
 
-#pragma message("WARNING: the 'dbg.h' header is included in your code base")
```

Comment:
why was this removed?

Reply:
I guess it's because it can get annoying getting that message in every compilation.

Reply:
#37

Reply:
Yeah I have to add it again.